expo/gbe/templatetags/gbe_tags.py

```diff
@@ -29,9 +34,9 @@ def volunteer_schedule(profile):
     events = profile.volunteer_schedule()
     schedule = [
         {'event': str(event),
-         'time': "%s - %s" % (event.starttime.strftime("%a, %I:%M %p"),
-                              (event.starttime + event.duration).strftime(
-                                  "%a, %I:%M %p")),
+         'time': "%s - %s" % (date_format(event.starttime, "DATETIME_FORMAT"),
+                              date_format(event.starttime + event.duration,
+                                  "DATETIME_FORMAT")),
```

Comment:
This does "Feb. 10, 2017, noon - Feb. 10, 2017, 4 p.m."
Could you modify it to be "Feb. 10, 2017, noon - 4 p.m.

not a blocker, since we didn't highlight this as a point for fixing... but while we're at it...